modules/power-monitoring-hardware-virtualization-support.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-hardware-virtualization-support_{context}"]
+= {PM-kepler} hardware and virtualization support
+
+{PM-kepler} is the key component of {PM-shortname} that collects real-time power consumption data from a node through one of the following power estimation methods:
+
+Kernel Power Management Subsystem (preferred)::
+* `rapl-sysfs`: This requires access to the `/sys/class/powercap/intel-rapl` host file.
+* `rapl-msr`: This requires access to the `/dev/cpu/*/msr` host file.
+
+The `estimator` power source::
+Without access to the kernel's power cap subsystem, {PM-kepler} uses a machine learning model to estimate the power usage of the CPU on the node.
++
+[WARNING]
+====
+The `estimator` feature is experimental, not supported, and should not be relied upon.
+====
+
+You can identify the power estimation method for a node by using the *Power Monitoring / Overview* dashboard.
+
+[WARNING]
+====
+{PM-shortname-c} works only in bare-metal deployments, because most public cloud vendors do not expose Kernel Power Management Subsystems to virtual machines.
```

Comment:
```suggestion
{PM-shortname-c} works only in bare-metal deployments. Most public cloud vendors do not expose Kernel Power Management Subsystems to virtual machines.
```

Reply:
IMO we just need to change this sentence, as they are just different related statements. One is not causing the other.